client/src/components/AddPerk.svelte

```diff
@@ -62,6 +70,10 @@
             <label for="expiryDate">Expiry Date</label>
```

Comment:
I can't remember, is this being stored correctly?

Reply:
I added a variable inside Perk class to track the expirydate